src/v1/providers/firestore.ts

```diff
@@ -52,38 +49,32 @@ export function document<Path extends string>(path: Path) {
   return _documentWithOptions(path, {});
 }
 
-/** @hidden */
 // Multiple namespaces are not yet supported by Firestore.
 export function namespace(namespace: string) {
   return _namespaceWithOptions(namespace, {});
 }
 
-/** @hidden */
 // Multiple databases are not yet supported by Firestore.
 export function database(database: string) {
   return _databaseWithOptions(database, {});
 }
 
-/** @hidden */
 export function _databaseWithOptions(
```

Comment:
IMO `_databaseWithOptions`, `_namespaceWithOptions`, `_documentWithOptions` can all be possibly removed as an export. `functions-builder` appears to use them, but we can clone-n-own in the separate file.

Thoughts

Reply:
Please mark this internal instead.

Let's try to keep the work to minimum - we don't necessarily have to fix things.

Reply:
Marked as internal it is!